client/components/sidebar.vue

```diff
@@ -250,21 +249,23 @@
       },
 
       createComment(highlightKey) {
-        if (highlightKey === this.commentToAdd.highlightKey) {
-          Comment.create(this.commentToAdd);
-          this.commentToAdd = null;
+        for (let i = this.$commentsToAdd.length - 1; i >= 0; i -= 1) {
```

Comment:
Why you go from back to front? Maybe add a comment.

Reply:
It could just be a while loop `while (this.$commentsToAdd.length)` and then you remove and create the first element with `Comment.create(this.$commentsToAdd.splice(0, 1))`.